languages/scala/reference/exercise-concepts/leap.md

```diff
@@ -0,0 +1,15 @@
+# Leap
+
+[Example implementation](https://github.com/exercism/scala/blob/master/exercises/leap/example.scala)
+
+## General
+
+- object or case class: used as class wrapper for exercise
+- functions: used as main entry point for the exercise
+- function arguments: input strands passed as arguments
+- return values: return a value from a function
+- divisible: `n % m == 0`  to test if `n` is divisible by `m`
+- `&&`: to combine two booleans using `AND`
```

Comment:
I think the `&&` and `||` operators can be grouped under one "Boolean operators" Concept.